tomcat/datadog_checks/tomcat/data/metrics.yaml

```
@@ -46,13 +46,24 @@ jmx_metrics:
             alias: tomcat.servlet.request_count
             metric_type: counter
     - include:
-        type: Cache
+        type: StringCache
         attribute:
           accessCount:
-            alias: tomcat.cache.access_count
+            alias: tomcat.string_cache.access_count
             metric_type: counter
-          hitsCounts:
-            alias: tomcat.cache.hits_count
+          hitCount:
+            alias: tomcat.string_cache.hit_count
+            metric_type: counter
+    - include:
+        # Example Bean: `Catalina:type=WebResourceRoot,host=localhost,context=/docs,name=Cache`
+        type: WebResourceRoot
+        name: Cache
+        attribute:
+          hitCount:
+            alias: tomcat.web.cache.hit_count
```

```suggestion
            alias: tomcat.web_cache.hit_count
```
To be consistant with the string cache maybe?

---

I think `.` is more suitable for this case since it's hierarchal. tomcat > web > cache.

In contrast StringCache is one specific class.